saleor/graphql/checkout/mutations/utils.py

```diff
@@ -45,6 +45,9 @@
 
 
 ERROR_DOES_NOT_SHIP = "This checkout doesn't need shipping"
+ERROR_CC_ADDRESS_CHANGE_FORBIDDEN = (
+    "Can't change shipping address for click and collect"
```

Comment:
Maybe we should inform here why it's forbidden? E.g., For Click&Collect delivery, the address is automatically set to the warehouse address and cannot be changed manually.